common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -201,6 +201,10 @@ cACTION_APMANAGER_WIFI_CREDENTIALS_UPDATE_REQUEST:
     _type: WSC::cConfigData
     _length: [ wifi_credentials_size ]
 
+cACTION_APMANAGER_START_WPS_PBC_REQUEST:
+  _type: class
+  bssid: sMacAddr
```

Comment:
I believe we just agreed to use interface names everywhere, so not ifindex, mac address, radio_uid or vap_id. In my view, "everywhere" includes the internal messages.

In other words, I think we should use hostap_iface here instead.

Reply:
removed